### PR TITLE
Improve readme.md (closes #1)

### LINKED ISSUE #1: Add readme.md to github project
Please add a short description with basic information like

- purpose of the project
- possible use cases
- how to build
- how to install
- status of development

I came from a link at https://www.kickstarter.com/projects/volla/volla-phone-designed-with-simplicity-and-security-in-mind/posts/2901919
and it just contained a "teaser" paragraph:

> In the context of our developer community an exciting project of „Luka" has progressed, that he started at the hackathon
> of last the Volla Community Days.
> On the Volla Phone it allows to choose between the start of two different operating systems, one of them installed on a fast SD card.
> This is not only exciting to try out an alternative operating system, but also to use a second one for use cases that need special security.

### PR DESCRIPTION
**Pull request by aryoda**
As announced here: https://github.com/Android-Boot-Manager/droidboot_device_volla_yggdrasil/issues/1#issuecomment-663950786

Note: This PR contains 3 commits (no separate feature branch, no rebase or merge squash into one commit since the PR target uses only one branch)

### CODE REVIEW

**Comment by nift4** [comment]
About How does the config file looks like? Which steps are required to add a new operating system? - you just use the Android(/VollaOS) App to do all that.

**Comment by nift4** [comment]
And there are merge conflicts

**Comment by aryoda** [comment]
The merge conflicts are due to this commit after forking:

https://github.com/Android-Boot-Manager/droidboot_device_volla_yggdrasil/commit/486e4c47c8fcddb28d729c929876a87d87aa9aa8

I can integrate these changes into my PR, but to solve the merge conflict you have to keep my changes as one ("take theirs") I guess.

Just a minute...

**Comment by aryoda** [comment]
> About How does the config file looks like? Which steps are required to add a new operating system? - you just use the Android(/VollaOS) App to do all that.

What exactly do you mean with "use the Android App"? Which App? I am aware of the menu you get when you switch off the phone to select entries like "switch off", "reboot", "reboot into recovery"...

**Comment by nift4** [comment]
> > About How does the config file looks like? Which steps are required to add a new operating system? - you just use the Android(/VollaOS) App to do all that.
> 
> What exactly do you mean with "use the Android App"? Which App? I am aware of the menu you get when you switch off the phone to select entries like "switch off", "reboot", "reboot into recovery"...

https://github.com/Android-Boot-Manager/App

**Comment by nift4** [comment]
The App README has mostly dead links tho

**Comment by nift4** [comment]
We didn't write docs yet ^^

**Comment by nift4** [comment]
We also have an Telegram group, https://t.me/andbootmgr - check it out to know more about the project

**Comment by aryoda** [comment]
OK, I have integrated the upstream changes to fix the merge conflict. I hope it is OK now...

File: README.md [hunk 1]
```diff
@@ -1,11 +1,59 @@
-# droidboot_device_volla_yggdrasil
-This repo contains lk based bootlader for the Volla Phone. It is based on sources from BSP. Created for dualbooting. It can display a simple GUI with lvgl, and read configs from an ext2 formated cache partion.
-First entry will call boot_linux_from_storage. 
+# Android boot manager for Volla phone (droidboot_device_volla_yggdrasil)
```
Comment (nift4) [inline README.md]: Please capitalize B and M of Android Boot Manager

**Review by nift4 — CHANGES_REQUESTED** [review]

**Review by nift4 — APPROVED** [review]
Thanks for this great README!

File: README.md (hunk 1 above)
Comment (aryoda) [reply README.md]: Done (I have also capitalized the P of Phone since this also done on the Volla Phone website)

**Pull request merged**